_sass/_base.scss

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
Didn't decide against it. I planned to add it in a separate PR. I'll add it here though since this is a PR about setting up some of the foundation.